openedx/core/djangoapps/content/block_structure/tasks.py

```diff
@@ -2,14 +2,22 @@
 Asynchronous tasks related to the Course Blocks sub-application.
 """
 import logging
+from capa.responsetypes import LoncapaProblemError
 from celery.task import task
 from django.conf import settings
+from edxval.api import ValInternalError
+from lxml.etree import XMLSyntaxError
 from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.exceptions import ItemNotFoundError
```

Comment:
I find it makes more sense to split into four levels:

# stdlib
# third party libs
# edx libs
# edx-platform imports

And to put a blank line between each group.  See the latest commit.